src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { User } from './types/User';
```

Comment:
Make sure that you pushed types files
<img width="309" alt="Screenshot 2022-09-26 at 12 01 20" src="https://user-images.githubusercontent.com/47606379/192236545-c61c5c5b-4bca-4686-8e15-9b9d6616d6d8.png">